src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -0,0 +1,90 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionList;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.VariableString;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static ch.njol.skript.lang.SkriptParser.ParseResult;
+
+@Name("Raw String")
+@Description("Returns the string without any formatting it nor stripping the formats from it, " +
+	"e.g. `raw \"&aHello There!\"` would output `&aHello There!`")
+@Examples("send raw \"&aThis text is unformatted!\" to all players")
+public class ExprRawString extends PropertyExpression<String, String> {
```

Comment:
Extend the SimplePropertyExpression and use its methods
```suggestion
public class ExprRawString extends SimplePropertyExpression<String, String> {
```

Reply:
I believe a PropertyExpression is better than a SimplePropertyExpression here, as the SimplePropertyExpression#getProperty() method would have the wrong formatting for this expressoin

Reply:
They are both the same except SimplePropertyExpression will make things easier by reducing the methods needed to override and the ability to use SimplePropertyExpression#convert() which should make things easier (it will handle the multiple strings, you will only have to deal with the expression as if it is a single string)

Also I think you mean `PropertyExpression#getExpr()` not `SimplePropertyExpression#getPropertyName()` because that's used for debugging

Reply:
SimplePropertyExpression wouldn't work in this case, as it gives you the string after it's actually formatted, so I wouldn't be able to get the unformatted string from the expression itself

Reply:
A SimplePropertyExpression should be okay here